.github/workflows/ci-images-garbage-collect.yaml

```
@@ -9,6 +9,7 @@ permissions: read-all
 
 jobs:
   scruffy:
+    if: github.repository == 'cilium/cilium'
```

```suggestion
    if: github.repository_owner == 'cilium'
```

nit: this probably could be just a check against the repo owner:

```
github.repository_owner == 'cilium'
```

(it would be good to do this check as a matter of course in more jobs, so we could [drop the doc guidance](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#clone-and-provision-environment) that suggests people disable Actions in their forks to avoid junk runs, this is probably not the only culprit)